src/main/java/org/traccar/protocol/Minifinder2ProtocolDecoder.java

```diff
@@ -165,21 +165,40 @@ protected Object decode(
                         if (BitUtil.check(alarm, 2)) {
                             position.addAlarm(Position.ALARM_FALL_DOWN);
                         }
+                        for (int i = 0; i < 4; i++) {
+                            if (BitUtil.check(alarm, i + 4)) {
+                                position.addAlarm(BitUtil.check(alarm, i + 26) ? Position.ALARM_GEOFENCE_ENTER : Position.ALARM_GEOFENCE_EXIT);
+                                position.set(Position.KEY_GEOFENCE, i + 1);
+                            }
+                        }
                         if (BitUtil.check(alarm, 8)) {
                             position.addAlarm(Position.ALARM_POWER_OFF);
                         }
                         if (BitUtil.check(alarm, 9)) {
                             position.addAlarm(Position.ALARM_POWER_ON);
                         }
+                        if (BitUtil.check(alarm, 10)) {
+                            position.addAlarm(Position.ALARM_MOVEMENT);
+                        }
+                        if (BitUtil.check(alarm, 11)) {
+                            position.addAlarm(Position.ALARM_IDLE);
```

Comment:
This doesn't look correct.

Reply:
Why not? 
Bit 10 is `Motion Alert` and bit 11 is `No-Motion Alert`.

Reply:
Ah got it. I don't think there is a suitable alarm for the `No-Motion Alert`, right?
So what do we do? Delete it for now?

Reply:
Delete it for now